### PR TITLE
Fix idlharness.js test for WebNN

### PR DESCRIPTION
**Pull request by foolip**

### CODE REVIEW

**Comment by foolip** [comment]
The error is now "FAIL message: promise_test: Unhandled rejection with value: object "TypeError: can't access property "createContext", navigator.ml is undefined" instead...

**Review by Honry — APPROVED** [review]
LGTM, thanks!

File: webnn/idlharness.https.any.js [hunk 1]
```diff
@@ -28,7 +28,7 @@ idl_test(
     });
 
     const operandType = {type: 'float32', dimensions: [1, 1, 5, 5]};
-    self.context = navigator.ml.createContext();
+    self.context = await navigator.ml.createContext();
```
Comment (Honry) [inline webnn/idlharness.https.any.js:31]: @foolip, `createContext()` is `sync` in [WebNN](https://webmachinelearning.github.io/webnn/#dom-ml-createcontext).

**Comment by BruceDai** [comment]
Thanks @foolip. 
Current I submit #35790 to update IDL tests based on latest WebNN API spec, PTAL, thanks.

**Comment by anssiko** [comment]
Superseded by now merged #35790

Closing this PR as discussed in https://www.w3.org/2022/12/01-webmachinelearning-minutes.html#t04

(Thanks @foolip for this contribution, we're actively working to improve the WebNN WPT coverage and may loop you in for your review for some other related PRs. This test suite comprises of tests for various ML-domain specific features so we've onboarded some additional ML experts to help with these WPT tests. Thanks for your help for the parts that fall on your area of expertise.)

**Pull request closed without merging**